compiler/test/dotty/tools/repl/TabcompleteTests.scala

```diff
@@ -138,4 +138,50 @@ class TabcompleteTests extends ReplTest {
       tabComplete("import quoted.* ; def fooImpl(using Quotes): Expr[Int] = { import quotes.reflect.* ; TypeRepr.of[Int].s"))
   }
 
+  @Test def wrongAnyMember: Unit =  fromInitialState { implicit s =>
```

Comment:
I didn't mean repeating the tests performed in `CompletionTest.scala` but rather directly reproducing the issue reported in https://github.com/lampepfl/dotty/issues/12514:
Starting with a fresh REPL try invoking completion on ``import scala.util.chaining.`s`` and then on ``import scala.util.chaining.`sc`` inside the same session